application/routes/user.py

```
@@ -57,12 +62,16 @@ def login():
             username = request.args['username']
             password = request.args['password']
+            logger.debug("User: Received login via GET for user: %s", username)
         else:
             username = request.json['username']
             password = request.json['password']
+            logger.debug("User: Received login via POST for user: %s", username)
 
         profiles.login(username, password)
+        logger.info("User: User logged in successfully: %s", username)
     except KeyError as e:
+        logger.error("User: Missing required field during login: %s", e)
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)